IngressMaxFields.user.js

```diff
@@ -13,6 +13,8 @@
 // @match http://www.ingress.com/intel*
 // @grant none
 // ==/UserScript==
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["$l" is defined but never used](https://www.codacy.com/app/stefan-nygren/IITC-Ingress-Maxfields-Exporter/file/1752266469/issues/source?bid=3039347&fileBranchId=3045939#l16)